src/main/java/org/opentripplanner/model/plan/ItinerariesCalculateLegTotals.java

```diff
@@ -8,12 +8,12 @@
  */
 class ItinerariesCalculateLegTotals {
 
-  int totalDurationSeconds = 0;
-  int transitTimeSeconds = 0;
+  Duration totalDuration = Duration.ZERO;
+  Duration transitTime = Duration.ZERO;
```

Comment:
Should we also rename these to `xDuration`?

Reply:
Already done in 3665a19f9f3ffcced8858ee71151a15f62cb972b